common/views/components/BasePage/BookPage.js

```diff
@@ -5,16 +5,35 @@ import BaseHeader from '../BaseHeader/BaseHeader';
 import Body from '../Body/Body';
 import HTMLDate from '../HTMLDate/HTMLDate';
 import Contributors from '../Contributors/Contributors';
+import PrimaryLink from '../Links/PrimaryLink/PrimaryLink';
 import {UiImage} from '../Images/Images';
 import WobblyBackground from '../BaseHeader/WobblyBackground';
+import {grid, spacing} from '../../../utils/classnames';
 import type {Book} from '../../../model/books';
 
 type Props = {|
-  book: Book
+  book: Book,
+  booksMetadataFlag: boolean
 |}
 
+const BookMetadata = ({book}: {| book: Book |}) => (
+  <dl className='grid'>
+    <dt className={'no-margin ' + grid({ s: 2, m: 2, l: 2, xl: 2 })}>Price</dt>
+    <dd className={'no-margin ' + grid({ s: 10, m: 10, l: 10, xl: 10 })}>{book.price}</dd>
+
+    <dt className={'no-margin ' + grid({ s: 2, m: 2, l: 2, xl: 2 })}>Format</dt>
+    <dd className={'no-margin ' + grid({ s: 10, m: 10, l: 10, xl: 10 })}>{book.format}</dd>
+
+    <dt className={'no-margin ' + grid({ s: 2, m: 2, l: 2, xl: 2 })}>Extent</dt>
```

Comment:
'Extent' seems like a weird heading, innit?

Reply:
Yea - we'll be working on the model over time as it's a bit crap.